src/recordlinker/linkage/models.py

```diff
@@ -47,3 +45,45 @@ class BlockingValue(Base):
     patient_id: orm.Mapped[int] = orm.mapped_column(ForeignKey("mpi_patient.id"))
     blockingkey_id: orm.Mapped[int] = orm.mapped_column(ForeignKey("mpi_blocking_key.id"))
     value: orm.Mapped[str] = orm.mapped_column(String(50), index=True)
+
+class Algorithm(Base):
+    __tablename__ = "algorithm"
```

Comment:
Is there a reason why this table name isn't prefixed with `mpi_`? Should we prefix with something else?

Reply:
I left it off because I was thinking of it as something different than an MPI. I can add it back just for consistency with the other tables unless we have any other ideas for the name?

Reply:
I've thought about this as well.  I think longer term we might have a structure like `models/mpi.py`, `models/config.py`, `models/training.py`, etc as we start to build out more features in the product.  At that time, distinct prefixes for each submodule makes sense.  I'm also fine with whats here now, and punting that decision to the future when we have more clarity.